### ISSUE TITLE
iOS app doesn't look good in landscape

### ISSUE THREAD

**Issue by hybridcattt** [tmrowco/electricitymap-contrib#3108]
Most of the map controls are unavailable, and the connectivity message becomes partially visible sometimes. The screenshot is from the App Store app, 1.2.3 v276, iPhone XS / iOS 14 (though the issue is not device- or OS version specific).

Perhaps the app should only support landscape on iPad, and not on iPhone? 

![1F78C33E-71D6-43A7-A1D2-C83598DABF2E](https://user-images.githubusercontent.com/11408899/115889429-39de2500-a454-11eb-8c5f-bf19c3cb181b.png)

**Comment by martincollignon** [comment]
Thanks for reporting @hybridcattt ! Feel free to tackle it ;)

**Comment by hybridcattt** [comment]
Might look at it at some point 😜

**Comment by corradio** [comment]
Related issue: https://github.com/electricitymaps/electricitymaps-contrib/issues/6966

**Comment by corradio** [comment]
@VIKTORVAV99 would it make sense to only support landscape iPad? I encounter similar issues when using my ipad for presentations.

**Comment by VIKTORVAV99** [comment]
> @VIKTORVAV99 would it make sense to only support landscape iPad? I encounter similar issues when using my ipad for presentations.

Yep and we are working on it, part of the problem are hardcoded sizes and the other part is the scaling.

But #7020 and then #6911 should help with this. Hopefully I can get both in a merge state this slack week.

**Comment by VIKTORVAV99** [comment]
I don't think we should exclusively allow it to be used in landscape or portrait though.

**Comment by madsnedergaard** [comment]
A lot has happened since this and we're now tracking issues like these internally, so I'll close this issue for now :)
